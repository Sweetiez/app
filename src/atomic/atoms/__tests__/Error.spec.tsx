import * as React from 'react';
import renderer from 'react-test-renderer';
import Error from '../Error';

describe('Error', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Error content={'test'} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
