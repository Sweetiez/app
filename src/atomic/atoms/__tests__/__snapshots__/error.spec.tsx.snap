// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#ff0000",
        "marginBottom": 10,
        "marginLeft": "auto",
        "marginRight": "auto",
      },
    ]
  }
>
  test
</Text>
`;
